lib/utils/ui/user_avatar.dart

```diff
@@ -0,0 +1,41 @@
+import "package:flutter/material.dart";
+
+/// A widget that displays the user's avatar.
+/// It provides a [onTap] parameter to handle the user's tap,
+/// which adds an InkWell response.
+class UserAvatar extends StatelessWidget {
+  const UserAvatar({
+    super.key,
+    required this.displayName,
+    this.radius,
+    this.onTap,
+  });
+
+  final String displayName;
+  final double? radius;
```

Comment:
Do think it is a good idea to have `radius` accept `null` as a parameter? 
When instanciating CircleAvatar, it will then default to 20 pixels (cf [here](https://api.flutter.dev/flutter/material/CircleAvatar/radius.html),) but it might be better to force the developer to set a radius, so that they follow the mockup.

Else we can also adapt the default size, for example:
`return CircleAvatar(
    radius: radius ?? (some condition ? value1: value2),
    ...
  );`

Reply:
You are completely right ! 

Following your comment, another solution would have been to use the default parameter as `this.radius = 20` in the constructor.

But I think that forcing the developer to carefully choose the radius according to the mockup and not just taking the default one is important.

Added in: [ec8f523](https://github.com/ProximaEPFL/proxima/pull/158/commits/ec8f523b7d68502d1d1cacf26a4c2091b0799645)

Thank you 😄